### DISCUSSION TITLE
403 error on request  listener:6000/api/v2/test-run/log-samples

### DISCUSSION THREAD

**Discussion by Michurink** [ludeknovy/jtl-reporter#257] [Q&A]
Hi!

I receive 403 error in Jmeter when trying to send results to listener. I check it  manually and receive the same error.
In listener logs:
{"level":30,"time":1705530290816,"pid":18,"hostname":"jtl-reporter-listener-f77dc645f-ksx58","reqId":"req-6","req":{"method":"POST","url":"/api/v2/test-run/log-samples","hostname":"dev-qa-jtl-reporter.dev.humans.uz:6000","remoteAddress":"10.100.29.104","remotePort":9615},"msg":"incoming request"}
{"level":30,"time":1705530290817,"pid":18,"hostname":"jtl-reporter-listener-f77dc645f-ksx58","reqId":"req-6","res":{"statusCode":403},"responseTime":0.6265640258789062,"msg":"request completed"}

What can be wrong and what could I check in  addition?

Thanks!

**Comment by ludeknovy** [comment]
Hi @Michurink ! Did you follow the tutorial here https://jtlreporter.site/docs/integrations/samples-streaming ? It looks like an issue with the token?
